pnpm-workspace.yaml

```diff
@@ -1,4 +1,12 @@
 packages:
-  - 'examples'
-  - 'packages/*'
-  - 'website'
+  - examples
+  - packages/*
+  - website
+
+onlyBuiltDependencies:
+  - '@swc/core'
+  - core-js
+  - core-js-pure
+  - es5-ext
+  - esbuild
+  - unrs-resolver
```

Comment:
Actually we don't want to allow these build scripts. I'm checking how we can drop this warning from pnpm while not showing the "error"